src/features/settings/Settings.tsx

```diff
@@ -27,6 +27,15 @@ export function Settings() {
   const [devMode, setDevMode] = useState(
     typeof devModeValue !== "undefined" ? devModeValue.value === "true" : false
   );
+  const unifiedEtaValue = settingsSelectors.selectById(
+    store.getState().settings,
+    "unifiedEta"
+  );
+  const [unifiedEta, setUnifiedEta] = useState(
+    typeof unifiedEtaValue !== "undefined"
+      ? unifiedEtaValue.value === "true"
+      : false
+  );
```

Comment:
is that possible to rewrite it with `??` ?

Reply:
nope